db/migrate/20250119222805_fill_fast_typing_threshold_based_on_deprecated_setting.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class FillFastTypingThresholdBasedOnDeprecatedSetting < ActiveRecord::Migration[7.2]
+  def up
+    old_setting_value =
+      DB.query_single(
+        "SELECT value FROM site_settings WHERE name = 'min_first_post_typing_time' LIMIT 1",
+      ).first
+
+    if old_setting_value.present?
+      fast_typing_threshold_setting =
+        case
+        when old_setting_value.to_i == NewPostManager::FAST_TYPING_THRESHOLD_MAP[:disabled]
+          "off"
+        when old_setting_value.to_i < NewPostManager::FAST_TYPING_THRESHOLD_MAP[:standard]
+          "low"
+        when old_setting_value.to_i < NewPostManager::FAST_TYPING_THRESHOLD_MAP[:high]
```

Comment:
We can't refer to these classes in migrations, it can cause issues later on if we remove them. Just hardcode the values here